conans/client/graph/range_resolver.py

```diff
@@ -78,14 +78,26 @@ def _search_remote_recipes(self, remote, search_ref):
             pattern_cached.update({remote.name: results})
         return results
 
+    @staticmethod
+    def _should_update(search_ref, update):
```

Comment:
If we eant to go ahead with this, we'll need to find a place to remove this duplication